docs/source/weights_loading.rst

```diff
@@ -176,6 +176,8 @@ you can restore the model like this
     # uses in_dim=128, out_dim=10
     model = LitModel.load_from_checkpoint(PATH, in_dim=128, out_dim=10)
 
+.. autofunction:: pytorch_lightning.core.lightning.LightningModule.load_from_checkpoint
```

Comment:
```suggestion
.. automethod:: pytorch_lightning.core.lightning.LightningModule.load_from_checkpoint
```

Hi @teddykoker, we need to use `automethod` to be able to show `[SOURCE]`
Please update pt_lightning_sphinx_theme locally to see the changes.